src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs

```diff
@@ -149,6 +149,14 @@ internal static partial class ControlDevel
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_EmitTextCursorMovedEvent")]
             public static extern global::System.IntPtr DaliAccessibilityEmitTextCursorMovedEvent(global::System.Runtime.InteropServices.HandleRef arg1, int arg2_pos);
 
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_GetSuppressedObjectEvents")]
+            public static extern uint DaliAccessibilityGetSuppressedObjectEvents(global::System.Runtime.InteropServices.HandleRef arg1);
+
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Accessibility_SetSuppressedObjectEvents")]
+            public static extern uint DaliAccessibilitySetSuppressedObjectEvents(global::System.Runtime.InteropServices.HandleRef arg1, uint suppressedObjectEvents);
```

Comment:
There should be a `void` return type here (it is `void` in `CSharp_Dali_Accessibility_SetSuppressedObjectEvents`).